README.md

```diff
@@ -18,6 +21,8 @@ Your group will be both a "Customer" and a "Designer".  First you'll prepare for
 
 ***Q1.1 Briefly describe your Mission:***
 
+A dog collar that alerts you when they leave your property. You can set 4 or more coordinates that form a specified area around the property. Collar alerts owner when dog outside the set property.
```

Comment:
I like the designated area idea. The ability to reset this area to match wherever you are would also be useful.